modules/mono/editor/bindings_generator.cpp

```diff
@@ -1607,7 +1607,7 @@ Error BindingsGenerator::_generate_cs_type(const TypeInterface &itype, const Str
 				   << INDENT1 "[DebuggerBrowsable(DebuggerBrowsableState.Never)]\n"
 				   << INDENT1 "private static readonly StringName "
 				   << CS_STATIC_FIELD_METHOD_NAME_PREFIX << imethod.name
-				   << " = \"" << imethod.name << "\";\n";
+				   << " = MethodName." << imethod.proxy_name << ";\n";
```

Comment:
It would be better if this field was removed entirely (it unnecessarily adds to the assembly size). The code that was using it should use `MethodName.proxy_name` directly.